build.ps1

```diff
@@ -205,6 +220,12 @@ function CreateZips([string] $runtimeSuffix) {
         Remove-Item $tempXdtDir -Recurse
     }
 
+    # Zip up symbols for builds with runtime embedded
+    if ($runtimeSuffix -eq  "") {
+        ZipContent "$buildOutput\publish.win-x86\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x86.zip"
+        ZipContent "$buildOutput\publish.win-x64\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x64.zip"
```

Comment:
If this continues to be a problem maybe we should pass the /DiasymreaderPath to crossgen.exe and see if it works? last time I tried, it had issues but might have been fixed by now.

 Debugging Parameters
    /CreatePDB <Dir to store PDB> [/lines [<search path for managed PDB>] ]
        When specifying /CreatePDB, the native image should be created
        first, and <assembly name> should be the path to the NI.
    /DiasymreaderPath <Path to diasymreader.dll>
        - Specifies the absolute file path to diasymreader.dll to be used.